charts/lfx-platform/values.yaml

```diff
@@ -146,6 +146,12 @@ opensearch:
 mailpit:
   enabled: true
 
+nack:
+  enabled: true
+  jetstream:
+    namespaced: true
+    nats:
+      url: nats://lfx-platform-nats.lfx.svc.cluster.local:4222
```

Comment:
this would need to be parameterized, right?

also, maybe URL is optional, fallback is the `-crd-connect` option?

https://github.com/nats-io/k8s/blob/main/helm/charts/nack/templates/deployment-jetstream-controller.yml#L93-L96

https://github.com/nats-io/nack/blob/d9c7e36fdde54e2d2f5d9a5d6437d66c098ab1f5/cmd/jetstream-controller/main.go#L73